### PR TITLE
Static Portfolio Page

### PR DESCRIPTION
**Pull request by Grace-Aka**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    display:flex;
+    flex-direction: column;
+    list-style-type:none;
+    padding: 0.8rem 1.4rem;
+    font-size: smaller;
+}
+
+ul li a{
+    text-decoration: none;
+    color:darkgrey;
```
Comment (Phildeoner) [inline style.css]: I will advise you change this color to something more visible like black or something of sort

File: style.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+h1{
+    font-size: 4rem;
+    color:maroon;
+    margin: 20px 0px 20px;
+}
+
+h3{
+    color: darkgrey;
```
Comment (Phildeoner) [inline style.css]: Use black or any darker color so it will be more visible

**Review by Phildeoner — COMMENTED** [review]
Hi grace! you did a great job, I have few suggestions
- Why not make your nav links to be inline instead of block I think it will be much better
- The main contents in the body why not align them to the center of the page? what do you think?

File: style.css (hunk 1 above)
Comment (Grace-Aka) [reply style.css]: Okay.. thanks

File: style.css (hunk 2 above)
Comment (Grace-Aka) [reply style.css]: Okay .. Thanks

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+
```
Comment (Phildeoner) [inline index.html:12]: Take out this space

File: index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            <h1>Hello I'm Grace Aka</h1>
+            <h3>I'm a Front-End Developer currently based in Nigeria. <br> I am motivated, detail-oriented and I make interactive <br> websites that are responsive on any device.
+            </h3>
+            <br>
+            <button class="btn">Portfolio</button>
+            <button class="btn2">Contact Me</button>
+        </div>
+
+        <div class="About">
+            <img src="Asset/ME.jpg" alt="A picture of me"></div>
```
Comment (Phildeoner) [inline index.html:34]: The closing div tag should be on line 35 not on the same line with the img

File: style.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    max-width: fit-content;
+    height: 2.5rem;
+    color: black;
+    font-weight: 200;
+    border: 5px dotted floralwhite;
+}
+
+.container{
+    align-items: center;
+}
```
Comment (Phildeoner) [inline style.css:21]: Add a space after this closing curly brace

**Review by Phildeoner — APPROVED** [review]
Hi @Grace-Aka👋 Great Job so far, Your work is complete but I will suggest you consider the comments below before merging so as to maintain a clean code👌
Happy Coding😊👍

File: style.css (hunk 5 above)
Comment (Grace-Aka) [reply style.css:21]: okay.. thanks

File: index.html (hunk 3 above)
Comment (Grace-Aka) [reply index.html:12]: okay

File: index.html (hunk 4 above)
Comment (Grace-Aka) [reply index.html:34]: okay

**Comment by SlimmyTee** [comment]
Well Done Grace

**Pull request merged**